stdlib/public/core/LazySequence.swift

```diff
@@ -128,7 +128,8 @@
 /// and discards the resulting array. Instead, use `reduce` for summing 
 /// operations, or `forEach` or a `for`-`in` loop for operations with side 
 /// effects.
-public protocol LazySequenceProtocol: Sequence {
+public protocol LazySequenceProtocol<Elements>: Sequence
+where Elements.Element == Element {
```

Comment:
Did you mean to add this? The constraint is already on the associatedtype below

Reply:
🤦‍♂️ No, this was left over from the previous version.